icu4c/source/data/makedata.mak

```diff
@@ -378,7 +387,7 @@ icu4j-data-install :
 #
 # testdata - nmake will invoke pkgdata, which will create testdata.dat
 #
-"$(TESTDATAOUT)\testdata.dat": "$(TESTDATA)\*" $(TOOLS_TS)
+"$(TESTDATAOUT)\testdata.dat": "$(TESTDATA)\*" $(TOOLS_TS) $(COREDATA_TS)
```

Comment:
Hm, this was part of another change and shouldn't have been this is PR...
Will update to remove.